RabbitMQ.AMQP.Client/Impl/AmqpQueueSpecification.cs

```diff
@@ -141,10 +141,7 @@ public IQueueSpecification Name(string queueName)
 
         public string QueueName
         {
-            get
-            {
-                return _queueName ?? string.Empty;
-            }
+            get { return _queueName ?? string.Empty; }
```

Comment:
Let's do this sort of formatting in a different PR. We can configure `dotnet format` to enforce a certain brace style.